scripts/README.md

```
@@ -11,58 +11,57 @@ Splits Grok mass-export files (with multiple conversations in a single JSON) int
 bun run split-conversations
 ```
 
-This will scan the translations directory and show what files would be split, but won't make any changes.
+This scans the translations directory and shows what files would be split, but makes no changes.
 
 **Apply changes:**
 ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)